### PR TITLE
Ma/client updates

### PR DESCRIPTION
**Pull request by markusahlstrand**
Trims all the application callback strings and updates the tenant clients on any changes in the migrations.

### CODE REVIEW

File: src/hooks/update-client.ts [hunk 1]
```diff
@@ -15,6 +15,13 @@ export async function updateTenantClientsInKV(env: Env, tenantId: string) {
   }
 }
 
+function splitUrls(value?: string) {
+  if (!value?.length) {
+    return [];
+  }
+  return value.split(",").map((key) => key.trim());
+}
```
Comment (danielgent) [inline src/hooks/update-client.ts:23]: I'd feel much more comfortable with unit tests but I suppose this function isn't critical path :smile: 

what was the issue then? White space in the urls set in auth-dashboard?

File: src/routes/tsoa/migrations.ts [hunk 2]
```diff
@@ -128,6 +128,8 @@ export class MigrationsController extends Controller {
       .where("id", "=", id)
       .execute();
 
+    await updateTenantClientsInKV(env, tenantId);
```
Comment (danielgent) [inline src/routes/tsoa/migrations.ts:131]: what is all this Markus?  I haven't seen the `migrations` route before. This isn't an OAuth standard is it?

**Review by danielgent — APPROVED** [review]
I'm not sure what this is fixing but if you're happy with it :smile:

File: src/routes/tsoa/migrations.ts (hunk 2 above)
Comment (markusahlstrand) [reply src/routes/tsoa/migrations.ts:131]: The migrations is a way of migrating passwords from auth0, connectId or wordpress, so it's not part of the standards. It's basically checking if the password is valid in another auth service and then adds it to our tenant. I was missing to update the client in KV-storage once the migrations in the SQL was updated, so it didn't have the latest version

**Pull request merged**